contracts/multisig-improved/src/ms_endpoints/propose_endpoints.rs

```diff
@@ -13,8 +13,14 @@ multiversx_sc::imports!();
 #[multiversx_sc::module]
```

Comment:
Unfortunately, "endpoints" is reserved by the framework, which is why I added the prefix.

Reply:
@andrei-marinica what's the situation where we have reserved folder names? Usually to frameworks have pre-defined working folders, such as we have `src`, `meta`, etc, but they come with the initial project so it's straight forward. For folders/files that are not part of the initial boilerplate (temporary generated code, etc), I think we could have prefixed names, such as `.endpoints` to avoid clashes, what do you say?